Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -11327,22 +11349,28 @@ command ideDoMessage pScript, pDebugMode
    local tScriptError
    try
       if pDebugMode then
-         # OK-2010-01-02: Remove the message box itself from the debug context. 
-         # Line 1 will be revDebugDoMessage
-         # Line 2 is revExecuteMessage (message box field)
-         # Line 3 is returnInField (message box field)
-         # Line 4 will hopefully be the user code (assuming of course the message box isnt' changed).
-         # Note that this code is called from both the single and multi-line fields, but luckily they both 
-         # have the same number of calls.
-         local tContext, tContextNumber
-         get revDebuggerContexts()
-         put line 4 of it into tContext
-         
+         local tContext, tScriptEditor
+         put revTopMostScriptEditor() into tScriptEditor
```

Comment:
@livecodeali I think this should this iterate open script editors for the topmost in debug mode. How about adding a `pMode` param to this function?